I-DOCSIS 7.0.3 (#3092)
SkylineCommunications/dataminer-docs: release-notes/EPM/I-DOCSIS/EPM_7.0.3.md

---
uid: EPM_7.0.3_I-DOCSIS
---

# EPM 7.0.3 I-DOCSIS (preview)

> [!IMPORTANT]
> We are still working on this release. Release notes may still be added, modified, or moved to a later release. Check back soon for updates!

## New features

#### EPM_I_DOCSIS_AddNewCcapCmPair improvements [ID_39450]

Several improvements have been implemented in the EPM_I_DOCSIS_AddNewCcapCmPair script, which is used to create a new CCAP.

- Every page now includes a *Cancel* button, which you can use the terminate the script.

> - Validation on fields now prevents the user from proceeding if fields still need to be filled in.

Reviewer: Hi,
Maybe we could rephrase this sentence? If I understand correctly the script now validates empty fields. If this is the case we can just say *The script now validates empty fields before to proceed with the provisioning*

Author: How about this?
```suggestion
- If fields that must be filled in are still empty, the field validation will now prevent the user from proceeding.
```

Reviewer: It looks ok. Thanks @MariekeGO!

- Every page now includes a *Cancel* button, which you can use the terminate the script.
- If fields that must be filled in are still empty, the field validation will now prevent the user from proceeding.
- When the script has been fully executed, a final page is now displayed with important information regarding the element creation process. If a message labeled as "[ERROR]" is shown in this window, this means that the creation of a specific element could not be completed. If a message labeled as "[INFO]" is shown, this message contains important information regarding the creation of elements.

## Changes

### Enhancements

#### Boundaries updated for US/DS thresholds [ID_39507]

For the following parameters in the upstream and downstream threshold tables, the threshold boundaries were updated:

- For the upstream QAM channel for the collector, the Minimum Tx Power Level is now 25.0 dBmV.
- For the upstream QAM channel for the CCAP, the Maximum Timing Offset Level is now 10000 µs.
- For the downstream QAM channel for the collector, the Maximum Rx Power Level is now 20 dBmV.

> 

Reviewer: Hi @sergiosuarezsky, can we change *collector* for *CM Collector*?

Reviewer: Updates were done on this review
